src/pages/Detail/components/DetailArticle/DetailArticle.jsx

```diff
@@ -1,38 +1,39 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
 import styled from 'styled-components';
+import SimpleSlider from './DetailSlider';
 
 export default function DetailArticle() {
+  const [detailData, setDetailData] = useState({});
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/DetailMockData.json')
+      .then(res => res.json())
+      .then(data => setDetailData(data[0]))
+      .then(setIsLoading(false));
+  }, []);
+
+  if (isLoading || !detailData) return <div>Loading</div>;
+
   return (
     <article>
-      <ArticleImage>
-        <img />
-      </ArticleImage>
-      <ArticleComments>
-        아이맥 사이로 촘촘하게 들어오는 아침해가 좋아서 얼른 아이폰 들고
-        찍었어요
-      </ArticleComments>
+      <SimpleSlider imageUrlList={detailData.image_urls} />
+      <ArticleComments>{detailData.content}</ArticleComments>
       <ArticleTags>
-        <span>#오하우스</span>
-        <span>#데스크테리어</span>
-        <span>#작업실</span>
+        {detailData.tags &&
+          detailData.tags.map((tag, index) => (
```

Comment:
```suggestion
        {detailData.tags?.map((tag, index) => (
```
이 부분도 옵셔널 체이닝으로 처리할 수 있습니다

Reply:
넵 반영하였습니다!